spec/serializers/api/declaration_serializer_spec.rb

```diff
@@ -66,13 +65,15 @@
             end
           end
 
-          context "when statement_item is the latest" do
+          context "when a linked statement item is moved to another statement" do
+            let!(:statement_item) { create(:statement_item, declaration:) }
+
             before do
-              declaration.statement_items.first.update!(updated_at: latest_datetime)
```

Comment:
could we freeze time here to a certain time and check against it as the tests usually set up things quite fast, and referring to the current time might pass anyway?